homeassistant/components/modbus/entity.py

```diff
@@ -256,16 +256,29 @@ def _swap_registers(self, registers: list[int], slave_count: int) -> list[int]:
             registers.reverse()
         return registers
 
-    def __process_raw_value(self, entry: float | str | bytes) -> str | None:
+    def __process_raw_value(
+        self,
+        entry: float | str | bytes,
+        scale: float | None = None,
+        offset: float | None = None,
+    ) -> str | None:
```

Comment:
this is an internal method, so why allow None ??? 

You already substitute None in the calling method.